hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -839,7 +853,9 @@ def get_mapping(self):
             "{}.Microscope Info.Voltage".format(tags_path): (
                 "Acquisition_instrument.TEM.beam_energy", lambda x: x / 1e3),
             "{}.Microscope Info.Stage Position.Stage Alpha".format(tags_path): (
-                "Acquisition_instrument.TEM.Stage.tilt_alpha", None),
+                "Acquisition_instrument.TEM.Stage.tilt_alpha", lambda x: round(x, 2)),
```

Comment:
Why using ``round`` here? Even if that's the current typical accuracy, this may change. Also, it would be good to list this enhancement in the PR description.

Reply:
Yeah, I though about it, but this already allows a precision of a 1/100 of a degree! It is just to avoid to display value with nonsense digits, but then, there is the question of why 2 and not 3 decimal digits...

Reply:
I would suggest not to round it to stay of the safe side...

Reply:
Done.